api/models/participant.py

```diff
@@ -2,10 +2,12 @@
 from django.conf import settings
 from django.core import mail
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
This is wide spread convention in Django approved by the documentation: https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#standard-translation